docs/tutorials/podman_tutorial.md

```diff
@@ -50,11 +50,11 @@ Note: The -l is a convenience argument for **latest container**.  You can also u
 of -l.
 
 ### Testing the httpd server
-Now that we have the IP address of the container, we can test the network communication between the host
+As we do not have the IP address of the container, we can test the network communication between the host
```

Comment:
OK, I'm confused by this change.  We have the IP of the container from the `podman inspect` call on line 44.  Then if you curl localhost, I think you'll get the IP of the host, not the container as we're not in the container at this point....

Reply:
The tutorial is meant to be run rootless, so the container does not have a valid IP. Since 8080 is forwarded to the container, localhost will work.